### PR TITLE
[VSCODE] Rewrote vscode preview plugin

### PR DESCRIPTION
**Pull request by tomlm**
* uses webpanel (from deprecated htmlPreview)
* uses published cdn renderer so it is always rendering using latest published renderer

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/3263)

### CODE REVIEW

File: source/vscode/src/adaptiveCardPanel.ts [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+                <script type="text/javascript" src="https://unpkg.com/markdown-it/dist/markdown-it.min.js"></script>
+                <script src="https://unpkg.com/adaptivecards/dist/adaptivecards.min.js"></script>
+            </head>
+            <body>
+                <h1>${filename} Preview</h1>
+                <div id='previewDiv' style="background-color:white;margin:8px 0px 20px 0px;"></div>
+                <script>
+                    var adaptiveCard = new AdaptiveCards.AdaptiveCard();
+					adaptiveCard.hostConfig = new AdaptiveCards.HostConfig({
+                        supportsInteractivity: true,
```
Comment (paulcam206) [inline source/vscode/src/adaptiveCardPanel.ts]: nitpick -- mixed whitespace here

File: source/vscode/src/extension.ts [hunk 2]
```diff
@@ -1,58 +1,26 @@
...
+export function activate(context: vscode.ExtensionContext) {
+    context.subscriptions.push(
+        vscode.commands.registerCommand('adaptivecardviewer.open', () => {
+            AdaptiveCardPanel.createOrShow(context.extensionPath);
+        })
+    );
+
+    if (vscode.window.registerWebviewPanelSerializer) {
+		// Make sure we register a serializer in activation event
+		vscode.window.registerWebviewPanelSerializer(AdaptiveCardPanel.viewType, {
```
Comment (paulcam206) [inline source/vscode/src/extension.ts]: nit: more mixed whitespace

File: source/vscode/src/test/extension.test.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+// Defines a Mocha test suite to group tests of similar kind together
+suite("Extension Tests", () => {
+
+	// Defines a Mocha unit test
+	test("Something 1", () => {
+		assert.equal(-1, [1, 2, 3].indexOf(5));
+		assert.equal(-1, [1, 2, 3].indexOf(0));
+	});
+});
```
Comment (paulcam206) [inline source/vscode/src/test/extension.test.ts]: is this scaffolding for future tests?

Comment (tomlm) [reply source/vscode/src/test/extension.test.ts]: yah, this was autogenerated by the template generator for plugins.  I have no idea how to actually unit test a vscode plugin 

---
In reply to: [315314829](https://github.com/microsoft/AdaptiveCards/pull/3263#discussion_r315314829) [](ancestors = 315314829)

**Comment by paulcam206** [comment]
closing out old PRs - feel free to revive this if it's still relevant

**Pull request closed without merging**